packages/react-html/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
```

Comment:
```suggestion
## [Unreleased]
```

Reply:
quilt convention tends towards not uncommenting / recommending this heading constantly